osu.Game.Rulesets.Osu/Edit/Blueprints/Spinners/SpinnerPlacementBlueprint.cs

```diff
@@ -29,22 +30,29 @@ protected override void Update()
         {
             base.Update();
 
+            if (isPlacingEnd)
+                HitObject.EndTime = Math.Max(HitObject.StartTime, EditorClock.CurrentTime);
+
             piece.UpdateFrom(HitObject);
         }
 
-        protected override bool OnClick(ClickEvent e)
+        protected override bool OnMouseDown(MouseDownEvent e)
         {
             if (isPlacingEnd)
             {
+                if (e.Button != MouseButton.Right)
+                    return false;
+
                 HitObject.EndTime = EditorClock.CurrentTime;
                 EndPlacement();
             }
             else
             {
-                isPlacingEnd = true;
-                piece.FadeTo(1f, 150, Easing.OutQuint);
```

Comment:
Intentionally removed?

Reply:
I actually removed it to figure out what it was doing, and never noticed a difference. It already starts at alpha 1, no?

Reply:
Should be 0.5? (ctor)

Reply:
ah, i see. i guess revert it if you think it looks better – removal was definitely not intentional.

Reply:
Added it back since without it there is no visual feedback that you're placing.